libmamba/include/mamba/solver/libsolv/unsolvable.hpp

```diff
@@ -21,7 +21,7 @@ namespace solv
 
 namespace mamba
 {
-    class Palette;
+    struct Palette;
```

Comment:
What was the warning regarding this?

Reply:
`Palette` is defined as `struct`: https://github.com/mamba-org/mamba/blob/af276aeedbb0f63a40ff500832425335d88029b9/libmamba/include/mamba/core/palette.hpp#L14

There is a warning when declaration and definition are different - as far as I remember in some ABIs (MSVC, if I remember correctly) this means different mangling and could cause linking errors.